felix/config/config_params.go

```diff
@@ -962,6 +963,8 @@ func loadParams() {
 			}
 		case "cidr-list":
 			param = &CIDRListParam{}
+		case "string-slice":
+			param = &StringSliceParam{}
```

Comment:
As noted above, I think you need validation that the string values are valid interface patterns.  Perhaps add a validation regex to your new  `StringSliceParam` ?

Reply:
Could I pls choose to not implement this suggestion because `StringSliceParam` was named added as a generic helper function to parse the raw input string and return as string slice i.e. no affinity to any input like valid interface(s) thus not subject to validation - the validation has been implemented above in the Felix config however which, if input there failed then, would not be need to be re-validated when set as config_params - thanks

Reply:
I do think you need validation here because Felix also supports env-var based configuration in manifest mode (and a config file, which is used in OpenStack).

I think the change here would be something like this:
```
case "interface-name-slice":
	param = &StringSliceParam{ValidationRegex: ...}
```
So your StringSliceParam could stay generic, but you'd have a particular sub-type with particular validation rules.

Reply:
Implemented string slice parameter validation as recommended